bokehjs/src/lib/models/renderers/glyph_renderer.ts

```diff
@@ -94,17 +99,20 @@ export class GlyphRendererView extends DataRendererView {
     if (hover_glyph != null)
       this.hover_glyph = await this.build_glyph_view(hover_glyph)
 
-    const {muted_glyph} = this.model
-    if (muted_glyph != null)
-      this.muted_glyph = await this.build_glyph_view(muted_glyph)
+    let {muted_glyph} = this.model
+    if (muted_glyph == null)
+      muted_glyph = mk_glyph({fill: {}, line: {}})
+    else if (muted_glyph == "auto")
+      muted_glyph = mk_glyph(muted_defaults)
+    this.muted_glyph = await this.build_glyph_view(muted_glyph)
```

Comment:
This block of code is repeated nearly verbatim several times now. What do you think about defining a helper function so that 
```
const {selection_config} = this.model
const selection_glyph =  construct_glyph(selection_config, selection_defaults)
this.selection_glyph = await this.build_glyph_view(selection_glyph)
```
is possible in  all cases? I think it could also be used for hover (the value will just never happen be "auto" in that case)

Reply:
I added `glyph_from_mode` in 85c93ee, which needs review. I'll resolve this conversation since it's outdated.